src/cmd/services/m3dbnode/config/config_test.go

```diff
@@ -357,6 +357,7 @@ func TestConfiguration(t *testing.T) {
     m3: null
     prometheus:
       handlerPath: /metrics
+      listenNetwork: ""
```

Comment:
nb; this is a new prom dependency that got pulled in on a glide update; think we're all good after this change, but can pin to that version if that's preferred